### PR TITLE
Automate packaging to build dist and publish to Pypi

### PR DESCRIPTION
**Pull request by quaxsze**

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
Thank you for this PR!

I have some comments on Makefile and Flit.

**Comment by maudetes** [comment]
After discussion, we've decided to keep on using CircleCI for udata modules for now, for coherence consideration.

**Review by maudetes — COMMENTED** [review]
Left to do:
- [ ] Find how to upload multiple `.devXXX` version on pypi with flit
- [ ] Remove trigger on ReleaseCI

File: gunicorn.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-gunicorn recherche:application -w 4 -b 0.0.0.0:5000
+gunicorn search:application -w 4 -b 0.0.0.0:5000
```
[the remark this answers is not in the document]
Comment (quaxsze) [reply gunicorn.sh:2]: It is used by the docker-compose file. But if this is not needed anymore (for local dev for example) we can.

[the remark this answers is not in the document]
Comment (maudetes) [reply gunicorn.sh]: Let's remove it and use only `flask run` for development

**Comment by quaxsze** [comment]
Approved ! 👍

**Pull request merged**